modules/sdk-core/src/bitgo/tss/ecdsa/ecdsa.ts

```diff
@@ -755,12 +755,12 @@ export async function verifyWalletSignature(params: {
   const publicUValueRawNotationIndex = 2 + params.verifierIndex;
 
   // Derive public form of u-value
-  const publicUValue = ecc.pointFromScalar(Buffer.from(params.decryptedShare.slice(0, 64), 'hex'), false);
+  const publicUValue = ecc.pointFromScalar(Buffer.from(params.decryptedShare.slice(0, 64), 'hex'), true);
```

Comment:
The notation data is being returned compressed - spotted this when testing using a local SDK build and examining the decrypted share.